src/index.js

```diff
@@ -58,13 +59,25 @@ function loader(...args) {
 }
 
 function pitch(remainingRequest, prevRequest, dataInput) {
-  const options = loaderUtils.getOptions(this) || {};
-  const cacheDirectory = options.cacheDirectory || path.resolve('.cache-loader');
+  const loaderOptions = loaderUtils.getOptions(this) || {};
+  const defaultOptions = {
```

Comment:
This is going to be the same for every call to `pitch()` so it would make sense to define it outside the fn.